client/my-sites/site-settings/redirect-to/README.md

```diff
@@ -0,0 +1,19 @@
+Check and restrict site access rights to the disconnection process.  
```

Comment:
Restrict rights only the disconnection process? I'd say we might want to use it for other Jetpack-only cases.

Reply:
Let's add a title to this README.